Versions 1.0.0+ are missing TypeScript support

<!-- Thanks for reporting an issue! Please fill out the blanks below. -->

## What are the steps to reproduce this issue?
1. Run any typescript project
2. Install honeybadger-js v1.0.0+

## What happens?
1. The file `honeybadger-js.d.ts` is missing, therefore, there are no types for `Honeybadger` instance.

## What were you expecting to happen?
Have types as they were until the version bump.

## Any logs, error output, etc?
```
error TS7006: Parameter 'notice' implicitly has an 'any' type.
types/lib.d.ts(58,31): error TS7016: Could not find a declaration file for module 'honeybadger-js'. '/finderly/Shpock-Web-3/node_modules/honeybadger-js/dist/honeybadger.cjs.js' implicitly has an 'any' type.
  Try `npm install @types/honeybadger-js` if it exists or add a new declaration (.d.ts) file containing `declare module 'honeybadger-js';`
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @shpock/web@3.0.0 check: `tsc --noEmit`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @shpock/web@3.0.0 check script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -4,6 +4,8 @@ CHANGELOG](http://keepachangelog.com/) for how to update this file. This project
 adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Fixed
+- Add TypeScript type definitions file to package.json
 
 ## [1.0.1] - 2019-04-16
 ### Fixed

--- a/package.json
+++ b/package.json
@@ -53,7 +53,8 @@
   },
   "readmeFilename": "README.md",
   "files": [
-    "dist"
+    "dist",
+    "honeybadger-js.d.ts"
   ],
-  "typings": "./honeybadger-js.d.ts"
+  "types": "./honeybadger-js.d.ts"
 }

